iaf-commons/src/main/java/nl/nn/adapterframework/util/FieldNameSensitiveToStringStyle.java

```diff
@@ -0,0 +1,93 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.util;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+/**
+ * Similar to the {@link ToStringStyle#SHORT_PREFIX_STYLE} which excludes
+ * fields containing the words 'password' and 'secret'.
+ * 
+ * @author Niels Meijer
+ */
+class FieldNameSensitiveToStringStyle extends ToStringStyle {
+	private static final List<String> PROTECTED_FIELDS = Arrays.asList("password", "secret");
```

Comment:
Do we want to filter for (oauth) tokens as well? Not sure if that's used anywhere, but logging a token for a logged in user might be sensitive as well

Reply:
If you have a complete list of potential fields to ignore... 😄
I doubt tokens matter, since it's usually bound to a session but we could add it to the ignore list.